vms/platformvm/service.go

```diff
@@ -99,6 +99,20 @@ func (s *Service) GetHeight(r *http.Request, _ *struct{}, response *api.GetHeigh
 	return err
 }
 
+func (s *Service) GetProposedHeight(r *http.Request, _ *struct{}, reply api.GetHeightResponse) error {
```

Comment:
```suggestion
func (s *Service) GetProposedHeight(r *http.Request, _ *struct{}, reply *api.GetHeightResponse) error {
```
This needs to be a reference parameter, no?